src/coredump/coredumpctl.c

```diff
@@ -938,9 +949,15 @@ static int run_debug(int argc, char **argv, void *userdata) {
                         arg_debugger = "gdb";
         }
 
-        debugger = strdup(arg_debugger);
-        if (!debugger)
-                return -ENOMEM;
+        r = strv_extend(&debugger_call, arg_debugger);
+        if (r < 0)
+                return log_oom();
+
+        if (arg_debugger_args) {
```

Comment:
actually the conditionalization here is unnecessary. all our strv_xyz() calls understand a NULL arrays as equivalent to an empty array, hence, strv_extend_strv() with the second argument being NULL is actually a safe NOP.